Dockerfile

```diff
@@ -114,4 +114,6 @@ USER root
 RUN pip3 install jsonschema PyYaml productmd requests && pip3 install --no-dependencies . && pip3 cache purge
 USER 1001
 
+RUN python3.12 -m pipx install --pip-args='--no-cache-dir' git+https://github.com/konflux-ci/pipeline-migration-tool.git
```

Comment:
Sorry I meant if you could add it to line 88:
```RUN pipx install --python python3.12 poetry pdm pipenv hashin uv hatch pip-tools git+https://github.com/konflux-ci/pipeline-migration-tool.git && rm -fr ~/.cache/pipx && pip3.12 cache purge```

and then remove the change in PATH

Reply:
Updated.  Any idea about the EC failures?

Reply:
I think it might be the current outage, let's wait a bit.